### PR TITLE
[Jess] Add gradle and unit test

### PR DESCRIPTION
**Pull request by JessicaHu03**
This pull request adds gradle build tool to the project.

Installed the following dependencies:
* Jackson core

A sample entry point to the program is also added for testing purposes.

### CODE REVIEW

**Comment by JessicaHu03** [comment]
Pushed some incorrect changes, doing a fix now. DO NOT MERGE

**Review by sandsieve — CHANGES_REQUESTED** [review]
Gradle adding approved. However there are still some conflict between the current tests and the older tests, need to resolve conflict before merging.

**Review by sandsieve — APPROVED** [review]

**Pull request merged**
